src/cephadm/build.py

```diff
@@ -26,6 +26,15 @@
 log = logging.getLogger(__name__)
 
 
+_VALID_VERS_VARS = [
+    "CEPH_GIT_VER",
+    "CEPH_GIT_NICE_VER",
+    "CEPH_RELEASE",
```

Comment:
what are we doing with this particular `CEPH_RELEASE` var? I see it's the only one here not used in the `command_version` function

Reply:
Good question, the cmake script passes it in as the number (eg. 18) but the code I coped from `ceph.in` doesn't make use of it. I would leave it as it gives us future flexiblity about what command_version prints, but I'm not wedded to the idea.